tools/test

```diff
@@ -192,12 +192,12 @@ check_node() {
     local node_version
     node_version="$(node --version)"
     case "${node_version}" in
-        v1[0-9].*) ;;
+        v[1-9][0-9].* | v[1-9][0-9][0-9].*) ;;
```

Comment:
`v[1-9][0-9].* | v[1-9][0-9][0-9]*` handles the v1000 problem.

But there’s a more standard solution: [`engines`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) in `package.json`.

Reply:
Great, yeah, in reality it works. If I set it to `>=20`:
```
$ time yarn install
yarn install v1.22.17
[1/5] Validating package.json...
error ZulipMobile@1.0.0: The engine "node" is incompatible with this module. Expected version ">=20". Got "14.18.3"
error Found incompatible module.
```

(I guess these docs are only claiming to document `npm` anyway, not `yarn`. Trying `npm install`, it seems to hang -- or maybe just takes a long time without producing much output -- and I don't feel like digging into it.)